etc/nginx/snippets/rewrites.conf

```diff
@@ -3,14 +3,14 @@
 set $current_version_sdk_c_api '3.3.2';
 set $current_version_sdk_dotnet_api '3.3.4';
 set $current_version_sdk_go_api '2.5.2';
-set $current_version_sdk_java_api '3.3.3';
+set $current_version_sdk_java_api '3.3.4';
 set $current_version_sdk_jvm_core_api '2.3.3';
```

Comment:
jvm_core should get a version update at the same time?

Reply:
`2.3.4` - yeah let me update 👍